### DISCUSSION TITLE
Downgrade To an older version

### DISCUSSION THREAD

**Discussion by Iman-Sh** [wg-easy/wg-easy#963] [General]
Hey everyone, how can i downgrade wg-easy to and older version or install a version other than the last one available.

**Comment by pheiduck** [comment]
Easy, just use a version tag like `ghcr.io/wg-easy/wg-easy:11` replace `11` with the Version you want to use. Checkout: https://github.com/wg-easy/wg-easy/pkgs/container/wg-easy/versions?filters%5Bversion_type%5D=tagged

1.0 should not be used as it was a build during development.
But keep in mind that older versions does not get any attention for security or other updates.
